MovieList/MovieList/Favourites/PersistenceManager.swift

```diff
@@ -9,46 +9,48 @@ import Foundation
 
 class PersistenceManager {
     
-    static private let defaults = UserDefaults.standard
+    let defaults = UserDefaults.standard
     
     enum Keys {
         static let favourites = "favourites"
     }
     
-    var favourites: [MovieResult] = [] {
-        didSet {
-            save()
-        }
-    }
+    var persistedFavourites = [String]()
     
-    func getFavourites() {
+    func getFavourites() -> [String] {
         guard
-            let favouritedMovies = PersistenceManager.defaults.object(forKey: Keys.favourites) as? Data,
-            let favourites = try? JSONDecoder().decode([MovieResult].self, from: favouritedMovies)
-        else { return }
-        self.favourites = favourites
+            let favouritedMovies = defaults.object(forKey: Keys.favourites) as? Data,
+            let favouritesArray = try? JSONDecoder().decode([String].self, from: favouritedMovies)
+        else { return [] }
+        self.persistedFavourites = favouritesArray
+        return self.persistedFavourites
     }
     
-    func addToFavourites(movie: MovieResult) {
-        let newFavourite = MovieResult(trackName: movie.trackName, artworkUrl100: movie.artworkUrl100, primaryGenreName: movie.primaryGenreName, longDescription: movie.longDescription, releaseDate: movie.releaseDate, previewUrl: movie.previewUrl)
-        
-        guard !favourites.contains(where: { movie in
-            favourites.append(newFavourite)
-            return true
-        }) else { return }
+    func addToFavourites(name: String) {
```

Comment:
Possibly rename to: `func saveMovieNameToFavourites(newMovie: String)`

Reply:
Renamed to: func saveMovieNameToFavourites(newMovie: String).